testingfacade/APIs/schemas/AnswerSchema.json

```diff
@@ -0,0 +1,45 @@
+{
+    "title": "Controller Testing - POST request to testing tool",
+    "description": "Contains answers to a question to be verified by the test suite",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "required": [
+      "name",
+      "answer_response",
+      "time_answered"
+    ],
+    "properties": {
+      "name": {
+        "description": "Question_id sent to Question API",
+        "type": "string"
+      },
+      "answer_response": {
+        "description": "Answer or answers to be verified",
+        "anyOf": [
+          {
+            "description": "Multi choice response",
+            "type": "array",
+            "items": 
+              {
+                "type": "string"
+              },
+            "examples": [
+              ["answer_1", "answer_3"]
+            ]
+          },
+          {
+            "description": "Single choice response",
+            "type": "string"
+          },
+          {
+            "description": "Action response",
+            "type": "null"
+          }
+        ]
+      },
+      "time_answered": {
+        "description": "Currently time in s from unix epoch that answer was sent",
```

Comment:
"Currently time"?

Reply:
That was a note in case it was going to change. Looking into your suggestion elsewhere to have it detected by the test suite, from my quick look yesterday I think it may have actually been implemented a while ago making this redundant.

Reply:
@hrbriggs Thanks, Heather! I was just saying to Jonathan that it's got to the point I can't find all the previous comments, sorry for the déja vu!

Reply:
@garethsb It's a completely bizarre system isn't it!